packages/experimental/Shadow/SPEC.md

```diff
@@ -47,3 +53,4 @@ If `undefined` is passed into the `shadowToken` prop, no shadow will be rendered
 ## Notes
```

Comment:
Could you add a `## Details / Implementation` (not sure which name I prefer  section where we briefly explain how Shadow adds an extra View around your view for the ambient shadow, and smartly extracts your style props so that the extra view and the child are exactly the same size? Maybe we can also add a bullet point about how in our investigation, Apple platforms needed their `blur radius` divided by 2 in comparison with web.